hive/Dockerfile

```diff
@@ -1,3 +1,34 @@
+# syntax=docker/dockerfile:1
+FROM stackable/image/java-base AS builder
+
+ARG PRODUCT
+
+RUN microdnf update && \
+    microdnf install \
+    maven \
+    tar \
+    gzip \
+    zip && \
+    microdnf clean all
+
+USER stackable
+WORKDIR /stackable
+
+SHELL ["bash", "-euo", "pipefail", "-c"]
+
+RUN curl --fail -L "https://repo.stackable.tech/repository/packages/hive/hive-${PRODUCT}-src.tar.gz" | tar -xzC .
+RUN mv hive-${PRODUCT} hive-${PRODUCT}-src
+RUN chmod +x patches/apply_patches.sh
```

Comment:
Not sure this line is needed if it is checked into the repo with execute permissions.